src/stores/cluster/index.js

```diff
@@ -16,14 +16,18 @@ export default class ClusterStore extends Store {
   @observable clusterCount = 0;
   @observable totalNodeCount = 0;
   @observable isModalOpen = false;
+  @observable modalType = '';
 
   @observable clusterId; // current delete cluster_id
   @observable operateType = '';
 
-  @observable currentPage = 1;
+  @observable currentPage = 1; //cluster table query params
   @observable searchWord = '';
+  @observable defaultStatus = ['active', 'stopped', 'ceased', 'pending', 'suspended'];
```

Comment:
`defaultStatus` no need to be set as observable

Reply:
ok, delete @observable .